css/join.css

```diff
@@ -0,0 +1,160 @@
+* {
+  --theme-background: #1d1283;
+  --text-white: #ffffff;
+  --text-red: #fd0101;
+  --input-background: #d9d9d9;
+  --error-border: #ff0000;
+  --error-background: #ffebef;
+  --correct-border: #09a444;
+  --correct-background: #ceedda;
+}
+
+.greeting {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+  padding: 0;
+  margin: 1.5rem 0;
+}
+
+.container {
+  width: max-content;
+  margin: 0 auto;
+  max-width: 35rem;
+}
+
+.container ol li {
+  line-height: 2rem;
+}
+
+.progress {
+  font-family: 'Roboto';
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 21px;
+  margin: 0 0 2rem;
+}
+
+.info,
+.input-label,
+.input-label-dark {
+  font-family: 'Roboto';
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.input-label {
+  font-weight: 400;
+}
+
+.input-label-dark {
+  font-weight: 500;
+}
+
+.user-input {
+  font-family: 'Roboto';
+  font-weight: 300;
+  font-size: 16px;
+  background-color: var(--input-background);
+  width: 30rem;
+  min-height: 2.5rem;
+  line-height: 2rem;
+  border: none;
+  border-radius: 9px;
+  margin: 1rem 0;
+  padding: 0 0.5rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.user-input-big {
+  font-family: 'Roboto';
+  font-weight: 300;
+  font-size: 16px;
+  background-color: var(--input-background);
+  width: 30rem;
+  height: 6.25rem;
+  line-height: 2rem;
+  border: none;
+  border-radius: 9px;
+  margin: 1rem 0;
+  padding: 0 0.5rem;
+  overflow-y: scroll;
+}
+
+.text-red {
+  color: var(--text-red);
+}
+
+.button-disabled {
+  background-color: var(--input-background) !important;
+  border: none;
+  color: var(--theme-background) !important;
```

Comment:
What is the reason behind putting '!important' here?

Reply:
because I am overwriting the styling for filled button